examples/gno.land/p/nt/commondao/commondao.gno

```diff
@@ -139,6 +139,8 @@ func (dao *CommonDAO) Propose(creator std.Address, d ProposalDefinition) (*Propo
 		return nil, ErrOverflow
 	}
 
+	// doesn't check if the prop creator is a dao member? i assume it's left  to the user of the p/ to define the behavior?
```

Comment:
Yes, the idea is that devs should explicitly check it.

Following that idea it would be good that the fact is mentioned in the `Propose()` method docs.
Documentation is something that should be improved 👍 

Not checking that the creator address is a DAO member makes the proposing open to other use cases where devs might want to create proposals without enforcing membership.